doc/apiref.rst

```diff
@@ -681,6 +681,58 @@ allowed in object keys.
    -1 if *key* was not found. The reference count of the removed value
    is decremented.
 
+.. function:: json_t *json_object_getn(const json_t *object, const char *key, size_t key_len)
+
+   .. refcounting:: borrow
+
+   Get a value corresponding to *key* no more *key_len* chars from *object*. Returns *NULL* if
+   *key* is not found and on error.
+
+   .. versionadded:: 2.13
+
+.. function:: int json_object_setn(json_t *object, const char *key, size_t key_len, json_t *value)
+
+   Set the value of *key* to *value* in *object*. *key* must be a
+   valid null terminated UTF-8 encoded Unicode string. If there
```

Comment:
The pull-request description says this is for non-null-terminated byte strings?

And above you say "*key* no more *key_len* chars", do you really mean "*key* exactly *key_len* chars"?

Reply:
Fixed, thanks!